pkg/ccl/backupccl/generative_split_and_scatter_processor.go

```diff
@@ -51,7 +51,14 @@ type generativeSplitAndScatterProcessor struct {
 	spec    execinfrapb.GenerativeSplitAndScatterSpec
 	output  execinfra.RowReceiver
 
-	scatterer splitAndScatterer
+	// scatterers contain the splitAndScatterers for the first group of split and
+	// scatter workers. Each worker needs its own scatterer as one cannot be used
+	// concurrently.
+	scatterers []splitAndScatterer
```

Comment:
What do you think about calling this one `chunkSplitAndScatterer` and in the comment indicating that these are used to split and scatter importSpanChunks?

Then the one below can be called `chunkEntrySplitAndScatterer` or `entrySplitAndScatterer`? It doesn't perform scatters so maybe we mention that in the comment and describe what it does.

Reply:
done, renamed and added the comments.